H2TauTau/python/proto/analyzers/TauIsolationCalculator.py

```diff
@@ -51,4 +51,26 @@ def process(self, event):
             tau.puppi_iso03_pt = sum(c_p.pt()*c_p.puppiWeight() for c_p in puppi_iso_cands_03)
             # Add puppi isolation
 
+            self.tauIsoBreakdown(tau)
+            
         return True
+
+
+    def tauIsoBreakdown(self, tau):
```

Comment:
not clear to me why this is needed at all (why aren't the original accessors fine?) and why you define this in two files instead of one